### ISSUE TITLE
it seems warmup scheduler has bug?

### ISSUE THREAD

**Issue by sHaol0w** [alibaba/cascade-stereo#17]
In utils.py , warmupscheduler, last epoch should not be used like that.  scheduler.step() will not update it unless step(epoch)
And more important, the meaning of epoch is wrong in that logic. 

Thats my personal opinion.......And I really appreciate your open source code
thanks

**Comment by gxd1994** [comment]
@sHaol0w  Hi. thx.
![image](https://user-images.githubusercontent.com/15042705/86249334-bd870680-bbe1-11ea-8f09-89d514a05017.png)
last_epoch ->  last_step
